src/main/groovy/liquibase/harness/generateChangelog/GenerateChangelogTest.groovy

```diff
@@ -24,6 +24,7 @@ class GenerateChangelogTest extends Specification {
     @Shared
     UIService uiService = Scope.getCurrentScope().getUI()
     String testResourcesPath = System.getProperty("user.dir") + "/src/test/resources/"
+    String generatedResourcesPath = "/src/test/resources/"
```

Comment:
plz change to something like
String resourcesDirPath = "/src/test/resources/"
String resourcesDirFullPath = System.getProperty("user.dir") + resourcesDirPath